src/mac_cairo.m

```diff
@@ -96,7 +96,7 @@ - (void)drawRect:(NSRect)rect
   const CGSize sizePt = CGContextConvertSizeToUserSpace(context, sizePx);
 
   // Convert coordinates to standard Cairo space
-  CGContextTranslateCTM(context, 0.0, -sizePt.height);
+  CGContextTranslateCTM(context, 0.0, sizePt.height);
   CGContextScaleCTM(context, scale, -scale);
```

Comment:
welp.... shoulda looked at this commit first :)

Reply:
wonder if the -height is only needed for OSX v12.x.x.  Wouldn't be great to add an ifdef for that, but could it be?

Reply:
Wonder if these two lines of Y-axis flipping should be a `once` kind of deal.   cache a bool "initialized" member in the private view and call the flip one time?  maybe it's just flipping itself an odd amount of times when using mac SDK vX.Y.Z... just a thought.

Reply:
note to all: I tested out the "once" idea.  no go on that.

Reply:
It was a good idea, but yeah, I think if that was happening a whole lot of things would be much more broken everywhere because this is called when the context is entered.